src/v2/auth.rs

```diff
@@ -151,22 +167,15 @@ impl Client {
             }
         };
 
-        let mut req = self.build_reqwest(reqwest::async::Client::new().get(url.clone()));
-
-        if let Some(t) = token {
-            let bearer = format!("Bearer {}", t);
-            if let Ok(basic_header) = reqwest::header::HeaderValue::from_str(&bearer) {
-                req = req.header(reqwest::header::AUTHORIZATION, basic_header);
-            } else {
-                let msg = format!("could not parse HeaderValue from '{}'", bearer);
-                error!("{}", msg);
-                return Box::new(future::err(Error::from(msg)));
-            };
+        let req = self.build_reqwest(reqwest::async::Client::new().get(url.clone()));
+        let req = if let Some(t) = token {
+            req.bearer_auth(t)
         } else {
             debug!("is_auth called without token");
+            req
         };
 
-        trace!("Sending reqwest '{:?}'", req);
+        trace!("Sending reqwest to '{}'", url);
```

Comment:
s/reqwest/request/ I guess?

Reply:
Aren't they synonymous now in the context of Rust programs? :wink: